src/utils/amplitude.ts

```diff
@@ -43,5 +98,6 @@ export const logSidevisningSkolepenger = (side: string) => {
     side,
     team_id: 'familie',
     applikasjon: 'SP-soknadsdialog',
+    skjemanavn: 'Skolepenger',
```

Comment:
Kunne man erstatte strengen her med enumen du har laget? 
`ESkjemanavn.Skolepenger`

Reply:
For så vidt på de tre andre  også

Reply:
Bra catch, fikset i 7b82b02 :)